src/components/DelegateNoAccessModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import useLocalize from '@hooks/useLocalize';
+import CONST from '@src/CONST';
+import ConfirmModal from './ConfirmModal';
+import Text from './Text';
+import TextLink from './TextLink';
+
+type DelegateNoAccessModalProps = {
+    isNoDelegateAccessMenuVisible: boolean;
+    onConfirm: () => void;
+    delegatorEmail: string;
+};
+
+export default function DelegateNoAccessModal({isNoDelegateAccessMenuVisible = false, onConfirm, delegatorEmail = ''}: DelegateNoAccessModalProps) {
+    const {translate} = useLocalize();
+    const basicnoDelegateAccessPromptStart = translate('delegate.notAllowedMessageStart', {accountOwnerEmail: delegatorEmail});
+    const basicnoDelegateAccessHyperLinked = translate('delegate.notAllowedMessageHyperLinked');
+    const basicnoDelegateAccessPromptEnd = translate('delegate.notAllowedMessageEnd');
```

Comment:
```suggestion
    const noDelegateAccessPromptStart = translate('delegate.notAllowedMessageStart', {accountOwnerEmail: delegatorEmail});
    const noDelegateAccessHyperLinked = translate('delegate.notAllowedMessageHyperLinked');
    const noDelegateAccessPromptEnd = translate('delegate.notAllowedMessageEnd');
```

I don't think we need the `basic` here (and if we do needs to be camelCase)

Reply:
and update below accordingly